tests/support/server.tcl

```diff
@@ -95,9 +95,10 @@ proc kill_server config {
 
     # kill server and wait for the process to be totally exited
     send_data_packet $::test_server_fd server-killing $pid
-    catch {exec kill $pid}
     # Node might have been stopped in the test
+    # Send SIGCONT before SIGTERM, otherwise shutdown may be slow with ASAN.
     catch {exec kill -SIGCONT $pid}
+    catch {exec kill $pid}
```

Comment:
@oranagra continue https://github.com/redis/redis/pull/14217#discussion_r2228863907
ASAN intercept signal, so I guess that when we send SIGCONT after SIGTERM, it might start doing some work again, causing the process to close very buffering.
and I saw you said in https://github.com/redis/redis/pull/8552#issuecomment-785416899
I think there should be no side effects in changing the order of signal.

Reply:
i don't think i meant that the order doesn't matter. i think i meant that sending SIGCONT first makes more sense.
but if she reproduced it and it was working then i'm fine.
i don't understand the ASAN issue, but this order makes more sense, and if it fixes the problem, then great.